ember-cli-build.js

```diff
@@ -5,6 +5,10 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
     // Add options here
+    'empress-blog-netlify-cms': {
+      "netlify-config": {
+      }
+    }
```

Comment:
there is no need to put this in here. It's a bit strange but adding anything to `ember-cli-build.js` in an addon will only affect the dummy app of that addon 🙈 you can just delete this change and it won't affect anything 👍